packages/react-api/src/api/SQL.d.ts

```diff
@@ -1,9 +1,13 @@
 import { Credentials, ExecuteSQL } from '../types';
 
-export function executeSQL(
-  credentials: Credentials,
-  query: string,
-  opts?: {
-    format: '' | 'geojson'
-  }
-): ExecuteSQL;
\ No newline at end of file
+export function executeSQL({
+    credentials,
+    query,
```

Comment:
Once we're considering breaking the interface... should we rename 'query' into 'sql'? IMO "query" suggests that just 'SELECT' is supported...

Reply:
I used query because it's the name chosen at the apis and to keep coherence